sbndqm/dqmAnalysis/PMT/SBND/CMakeLists.txt

```
@@ -0,0 +1,37 @@
+simple_plugin( CAENV1730StreamsSBND "module"
+
+  art_Utilities
+  canvas
+
+  art_Framework_Services_Registry
+
+  ${LARDATAOBJ}
+  lardataobj_RawData
+  ${ART_UTILITIES}
+  ${FHICLCPP}
+  ${MF_MESSAGELOGGER}
+  ${MF_UTILITIES}
+  ${ART_FRAMEWORK_CORE}
+
+  ${ROOT_BASIC_LIB_LIST}
+
+  art_root_io_tfile_support
+  art_root_io_TFileService_service
+
+  sbndaq_artdaq_core::sbndaq-artdaq-core_Overlays_Common 
+
+  artdaq_core::artdaq-core_Utilities
+  artdaq_utilities::artdaq-utilities_Plugins
+
+  sbndaq_online_storage
+  sbndaq_online_redis_connection
+  sbndaq_online_hiredis
+
+  sbndqm_Decode_PMT_PMTDecodeData
+  tpcAnalysis_SBN
+
+  larana_OpticalDetector_OpHitFinder
```

I think  most of these Cmake variables are not set at all.
Judging from the module source code, a proper link list might look like:
```suggestion
  lardataobj::RawData
  larana::OpticalDetector_OpHitFinder
  sbndaq_online_storage
  sbndaq_online_redis_connection
  sbndaq_online_hiredis
  sbndqm::Decode_PMT_PMTDecodeData
  tpcAnalysis_SBN
```